src/geotop/deallocate.cc

```diff
@@ -113,7 +113,6 @@ void dealloc_all(TOPO *top,SOIL *sl,LAND *land,WATER *wat,CHANNEL *cnet,
     geolog << "Deallocating soil" << std::endl;
 
     free_doubletensor(sl->pa);
```

Comment:
why this one no?

Reply:
This hasn't been change yet. I had some trouble with ```pa``` so I'll end with all the other DOUBLETENSOR and then I'll let you know.

Regarding the logger I'll open another PR.

Thank you!